fedcloudclient/secret.py

```diff
@@ -32,24 +34,30 @@ def secret_client(access_token, command, path, data):
     :return: Output data from the service
     """
 
-    client = hvac.Client(url=VAULT_ADDR)
-    client.auth.jwt.jwt_login(role=VAULT_ROLE, jwt=access_token)
-    checkin_id = get_checkin_id(access_token)
-    full_path = checkin_id + "/" + path
-    function_list = {
-        "list_secrets": client.secrets.kv.v1.list_secrets,
-        "read_secret": client.secrets.kv.v1.read_secret,
-        "delete_secret": client.secrets.kv.v1.read_secret,
-    }
-    if command == "put":
-        response = client.secrets.kv.v1.create_or_update_secret(
-            path=full_path,
-            mount_point=VAULT_MOUNT_POINT,
-            secret=data,
-        )
-    else:
-        response = function_list[command](path=full_path, mount_point=VAULT_MOUNT_POINT)
-    return response
+    try:
+        client = hvac.Client(url=VAULT_ADDR)
+        client.auth.jwt.jwt_login(role=VAULT_ROLE, jwt=access_token)
+        checkin_id = get_checkin_id(access_token)
+        full_path = checkin_id + "/" + path
+        function_list = {
+            "list_secrets": client.secrets.kv.v1.list_secrets,
+            "read_secret": client.secrets.kv.v1.read_secret,
+            "delete_secret": client.secrets.kv.v1.read_secret,
```

Comment:
Should this be instead
```
"delete_secret":  client.secrets.kv.v1.delete_secret,
```
xref: https://hvac.readthedocs.io/en/stable/usage/secrets_engines/kv_v1.html#delete-a-secret

Reply:
Sure: https://github.com/tdviet/fedcloudclient/pull/154